src/Results/Result.php

```diff
@@ -23,14 +25,21 @@ class Result
     private $complexity;
 
     /**
+     * The config values.
+     * @var Config
+     */
+    private $config;
+
+    /**
      * Class constructor.
      * @param array $data Data to store in the object.
      */
-    public function __construct(array $data)
+    public function __construct(array $data, Config $config)
```

Comment:
Not sure if I like the entire `Config` object being stored in the results.

Wonder if it would be better off just providing the formula because that's all is needed.

The formula could be a part of the `$data` array.